template/script.sh.erb

```diff
@@ -349,6 +356,15 @@ jupyter kernelspec list
 { set +x; } 2>/dev/null
 echo "TTT - $(date)"
 
+# Setup custom Application directory so user can manage extensions
+<%- if extension_enabled -%>
+export JUPYTERLAB_DIR=$HOME/.jupyter/lab/<%= context.version.split('/').last.split('.')[0..1].join('.') %>
```

Comment:
Let's fix this up a bit so that it's more reliable because an expression like that may fail in some unexpected way. 

Let's put this into a function you can define in the `<%- -%>` block at the top 
```ruby
def jupyterlab_dir
  base = "#{OodSupport::User.new.dir}/.juypter/lab"
  match = context.version.match('^app_jupyter/(?<version>\d\.\d)')
  version = if !match.nil? && match.named_captures.key?('version')
              match[:version]
            else
              'unknown'
            end

  "#{base}/#{version}"
end
```

Then the template line here is just

```ruby
export JUPYTERLAB_DIR="<%= jupyterlab_dir %>"
```